### PR TITLE
Task view about me

### PR DESCRIPTION
**Pull request by rrrichard**
View About Me Task

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -54,8 +65,9 @@
         <a name="about"></a>
         <div>
             <h2>ABOUT ME</h2>
-            <p>My name is Richard and I am aspiring to become a full stack developer. I am currently studying at the Mayden Academy in Bath and have been recently certified as a Scrum Master.</p>
-            <p>In addition to my growing web development skills, I have a Bachelor of Arts degree in Architecture and have been working as an Architectural Technician for the last 3 years. During that time I have acquired skills that are similar in nature to being a developer as well as soft skills that makes me an excellent addition to a team. In my free time, I enjoy photography, cooking and creating digital artworks using architectural software.</p>
+<!--            <p>My name is Richard and I am aspiring to become a full stack developer. I am currently studying at the Mayden Academy in Bath and have been recently certified as a Scrum Master.</p>-->
```
Comment (HazDuck) [inline index.php:68]: probs delete these comments out now.

File: php/functions.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+/**
+ * this function selects the paragraphs in the database and fetches all the data
+ *
+ * @param $db string is the variable that connects the php to mysql and allows it to be called in functions
+ *
+ * @return mixed returns all the data in order of their id
+ */
+function addAboutMe($db)
```
Comment (HazDuck) [inline php/functions.php:11]: add PDO for the type hinting? e.g. (PDO $db)

File: php/functions.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+{
+    $query = $db->prepare("SELECT `paragraph` FROM `about_me`");
```
Comment (HazDuck) [inline php/functions.php:13]: add a WHERE in this to prevent deleted fields from being fetched from the DB?

File: php/functions.php [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ */
+function addParagraphs(array $paragraphs) :string {
+    if (is_array($paragraphs)) {
+
+        $paragraphPlaceholder = '';
+        foreach ($paragraphs as $paragraph) {
+            $paragraphPlaceholder .= '<p>' . $paragraph['paragraph'] . '</p>';
+        }
+        return $paragraphPlaceholder;
+    } else return 'You must enter an array';
```
Comment (HazDuck) [inline php/functions.php]: i think this requires {} around the return? e.g. else {return blah}

File: php/tests.php [hunk 5]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+require_once  'functions.php';
+
+use PHPUnit\Framework\Testcase;
+
+class FunctionTest extends Testcase
```
Comment (HazDuck) [inline php/tests.php:7]: do charlie's changes

File: php/tests.php [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    public function testAddParagraphsFailure3(){
+        $expected = '';
+        $input = [['awesome' => []]];
+        $case = addParagraphs($input);
+        $this->assertEquals($expected, $case);
+    }
+
+
+    public function testAddParagraphsMalformed()
+    {
```
Comment (HazDuck) [inline php/tests.php:39]: probs format this as the same as the rest with { on the above line

File: index.php (hunk 1 above)
Comment (rrrichard) [reply index.php:68]: woops i forgot about those

File: php/functions.php (hunk 3 above)
Comment (rrrichard) [reply php/functions.php:13]: done

File: php/functions.php (hunk 4 above)
Comment (rrrichard) [reply php/functions.php]: this was deleted because of charlie's changes

File: php/tests.php (hunk 6 above)
Comment (rrrichard) [reply php/tests.php:39]: gotcha

**Pull request merged**